system-int-test/test/cmr/system_int_test/ingest/variable_ingest_test.clj

```diff
@@ -93,15 +112,37 @@
                           update-grant-id
                           update-group-id
                           (ingest-util/get-ingest-update-acls update-token))))
-      (testing "create responses"
+      (testing "disallowed create responses:"
         (are3 [token expected]
           (let [response (variable-util/create-variable token variable-data)]
             (is (= expected (:status response))))
-          "System update permission allowed"
-          update-token 201
-          "Regular user denied"
-          reg-user-token 401
-          "Guest user denied"
+          "no token provided"
+          nil 401
+          "guest user denied"
+          guest-token 401
+          "regular user denied"
+          reg-user-token 401))
+      (testing "disallowed update responses:"
+        (are3 [token expected]
+          (let [update-response (variable-util/update-variable
+                                 token
+                                 (string/lower-case (:Name variable-data))
+                                 variable-data)]
+            (is (= expected (:status update-response))))
+          "no token provided"
+          nil 401
+          "guest user denied"
```

Comment:
@chris-durbin I've updated this branch with the first two -- I will update the delete branch with the latest two.